From ae0ba624c74bb03b03871958a2338b1f9752b73a Mon Sep 17 00:00:00 2001
From: pascal paillet <p.paillet@st.com>
Date: Fri, 28 Apr 2017 14:28:55 +0200
Subject: [PATCH] dt-bindings: regulator: describe regulator protection

describe how to bind the regulator protection consumer.

Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/204694
Change-Id: Id59410e64d90fdbbdb558352a5a52302014eb69d
---
 .../regulator/protection-consumer.txt         | 23 +++++++++++++++++++
 1 file changed, 23 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/regulator/protection-consumer.txt

diff --git a/Documentation/devicetree/bindings/regulator/protection-consumer.txt b/Documentation/devicetree/bindings/regulator/protection-consumer.txt
new file mode 100644
index 000000000000..bf8169e008c3
--- /dev/null
+++ b/Documentation/devicetree/bindings/regulator/protection-consumer.txt
@@ -0,0 +1,23 @@
+regulator protection bindings
+
+
+Required properties:
+	compatible: "protection-consumer"
+	protection-supply: the phandle of the regulator to control
+
+Optional properties:
+--------------------
+	interrupt
+
+
+Example:
+
+	regulator_protection {
+		compatible = "protection-consumer";
+		protection-supply = <&vdd>;
+		status = "okay";
+
+		interrupts = <15 2>;
+		interrupt-parent = <&gpiof>;
+	};
+
-- 
2.39.5

